gateway-ha/src/main/java/io/trino/gateway/ha/config/HaGatewayConfiguration.java

```diff
@@ -225,4 +231,16 @@ public void setManagedApps(List<String> managedApps)
     {
         this.managedApps = managedApps;
     }
+
+    public List<String> getStatementPaths()
+    {
+        return statementPaths;
+    }
+
+    public void setStatementPaths(List<String> statementPaths)
+    {
+        // remove trailing slashes to ensure predictable behavior when splitting on "/"
+        this.statementPaths =
+                ImmutableList.copyOf(Stream.concat(statementPaths.stream(), this.statementPaths.stream()).map(s -> s.replaceAll("/$", "")).toList());
```

Comment:
This appends the received argument to `statementPaths` each time the function is called. This is not an usual behavior for a setter. This function could potentially be called multiple times, eg. during testing.
We could simply set `statementPaths` to `V1_STATEMENT_PATH + <received argument>`.